### PR TITLE
Memo confirm scroll

### LINKED ISSUE #244: memo confirm runs offscreen
Screenshot from 2023-03-17 17-53-19

### PR DESCRIPTION
**Pull request by fluidvanadium**
fixes https://github.com/zingolabs/zingo-mobile/issues/244

yikes, it's been three weeks?? im sorry it took me so long to get around to this. i had to fix some other things

i changed display options so the scrollbar is visible, and scrolled the test

### CODE REVIEW

File: components/Send/components/Confirm.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Confirm: React.FunctionComponent<ConfirmProps> = ({ closeModal, confirmSen
             padding: 10,
             borderWidth: 1,
             borderRadius: 10,
-            borderColor: colors.border,
+            borderColor: "red",
```
Comment (juanky201271) [inline components/Send/components/Confirm.tsx]: I don't see why this box is changed to red....

File: components/Send/components/Confirm.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const Confirm: React.FunctionComponent<ConfirmProps> = ({ closeModal, confirmSen
         noSyncingStatus={true}
         noDrawMenu={true}
       />
-      <ScrollView contentContainerStyle={{ display: 'flex', justifyContent: 'flex-start' }}>
+      <ScrollView contentContainerStyle={{ display: 'flex', justifyContent: 'flex-start' }} showsVerticalScrollIndicator={true} persistentScrollbar={true} indicatorStyle={'white'} testID="send.scrollView">
```
Comment (juanky201271) [inline components/Send/components/Confirm.tsx]: A couple of things...
* Even with those props I cannot see the scrollbar in my Android Device, either in my emulator, I don't know what is happening.
* if there is a decision about change this behavior in this screen, I think we need to replicate it in all the screens that they have a `ScrollView` component... just for coherence.

**Review by juanky201271 — CHANGES_REQUESTED** [review]

**Comment by juanky201271** [comment]
@fluidvanadium can you put a screenshot of the result of this scrollbar???? thanks.

File: components/Send/components/Confirm.tsx (hunk 2 above)
Comment (fluidvanadium) [reply components/Send/components/Confirm.tsx]: 2) hmm, good point

File: components/Send/components/Confirm.tsx (hunk 1 above)
Comment (fluidvanadium) [reply components/Send/components/Confirm.tsx]: oops

**Comment by fluidvanadium** [comment]
1. the scrollView component renders in a dark color. it's difficult to see
changing this across both os requires adding a new module.
2. this is the only place that has needed to scroll yet. but, there are other scrollviews. maybe i should build a custom component and replace all scrollviews with it. this would facilitate (1.)

for now, this partly fixes the issue, and it also updates the test so it passes, which is necessary for e2e ci

a screenshot. the scroll bar is at the lower right
Screenshot from 2023-04-05 00-10-56

**Comment by juanky201271** [comment]
The only place where you can have the same problem: `scrolling memo` is in the detail of transactions... 

here the ScrollView: https://github.com/zingolabs/zingo-mobile/blob/da04bb79fcec031a831b3496f33214e34e92cf13/components/History/components/TxDetail.tsx#L81

and the memo field here: https://github.com/zingolabs/zingo-mobile/blob/da04bb79fcec031a831b3496f33214e34e92cf13/components/History/components/TxDetail.tsx#L200

I think in both places (confirm send & tx detail) is good to have the same logic.... because in both places you can have a really long memos...

**Comment by juanky201271** [comment]
@fluidvanadium If you are really busy I can help you with this PR, the change I asked you is really easy, if you want me to do it, let me know, I'm happy to help.

**Comment by juanky201271** [comment]
> @fluidvanadium If you are really busy I can help you with this PR, the change I asked you is really easy, if you want me to do it, let me know, I'm happy to help.

I'm happy to finish this PR if you want to... (I know you are very busy...)

**Comment by fluidvanadium** [comment]
hi juanky! i got this, dont worry

**Review by juanky201271 — APPROVED** [review]

**Pull request merged**